[TASK] Add hints to PSR-7 Request availibility in DataProcessors (#6562)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/RequestLifeCycle/Typo3Request.rst

        }
    }

..  hint::

>     Utilizing the PSR-7 request context inside a DataProcessor is subject

Author: ```suggestion
    Utilizing the PSR-7 request context inside a DataProcessor is
```

>     to rework in future TYPO3 versions. The `getRequest()` method is currently

Reviewer: ```suggestion
    to rework in future TYPO3 versions. The `$cObj->getRequest()` method is currently
```

>     to rework in future TYPO3 versions. The `$cObj->getRequest()` method is currently

Author: ```suggestion
    currently not attainable via proper API. The `$cObj->getRequest()` method is currently
```

>     still the recommended way to retrieve the reqeuest object, even though marked

Reviewer: ```suggestion
    still the recommended way to retrieve the request object, even though marked
```

    as `@internal`. It internally falls back to utilize the :php:`$GLOBALS['TYPO3_REQUEST']`

>     variable, which will vanish in the near future.

Reviewer: "near future" can be misleading, if this is available then in v15, v16, .... Anyway, does it really help to write about the implementation of an internal method?

The global variable is deprecated since v9 AFAIK ;-)

Author: The internal method is currently the only way to access it via API so hard to disrecommend it. I'll rephrase.

>     variable, which will vanish in the near future.

Author: ```suggestion
    variable, which could also be utilized directly (for now).
```
